apps/client/src/libs/data/songs.ts

```diff
@@ -36,7 +36,7 @@ export const songs: AniSong[] = [
       name: 'Dimension Tripper!!!!',
       author: 'nao',
       lyrics:
-        'そう今全ての次元が 結ばれる様に明日を飛び超えて まだ見えない新たな歴史を 描けるように…「Fortune favors the brave.」Dimension tripper!!!!'
+        'そう{今,いま}{全,すべ}ての{次,じ}{元,げん}が {結,むす}ばれる{様,よう}に{明,あ}{日,ひた}を{飛,と}び{超,こ}えて まだ{見,み}えない{新,あら}たな{歴,れき}{史,し}を {描,えが}けるように…「Fortune favors the brave.」{Dimension,ディメンション} {Tripper,トリッパー}!!!!'
```

Comment:
why Dimension Tripper have ふりがな when Fortune favors the brave don't 💀 
```suggestion
        'そう{今,いま}{全,すべ}ての{次,じ}{元,げん}が {結,むす}ばれる{様,よう}に{明,あ}{日,した}を{飛,と}び{超,こ}えて まだ{見,み}えない{新,あら}たな{歴,れき}{史,し}を {描,えが}けるように…「Fortune favors the brave.」Dimension Tripper!!!!'
```

Reply:
Commit first, approve later lol